ouroboros-network-api/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -122,7 +122,7 @@ instance Acceptable NodeToClientVersionData where
 instance Queryable NodeToClientVersionData where
     queryVersion = query
 
-nodeToClientCodecCBORTerm :: NodeToClientVersion -> CodecCBORTerm Text NodeToClientVersionData
+nodeToClientCodecCBORTerm :: ntcVersion -> CodecCBORTerm Text NodeToClientVersionData
 nodeToClientCodecCBORTerm _v = CodecCBORTerm {encodeTerm, decodeTerm}
```

Comment:
Do we still need this change?

Reply:
Hmm, we might need to introduce `Cardano` name-space in `ouroboros-network-api`.

Reply:
Yeah I don't think we need to go via that route just yet, the fields are quite independent/agnostic from Cardano Node for now. I'll revert this diff